lib/index.js

```diff
@@ -1,19 +1,41 @@
 var foodsRequests = require('./requests/foods')
+var foodsDiary = require('./requests/diaryFoods')
 require('./foods')
 
 $(document).ready(() => {
-  foodsRequests.getFoods()
+  var fileName = location.pathname.split('/').slice(-1)[0]
+
+  if (fileName === 'foods.html') {
+    foodsRequests.getFoods()
+  } else {
+    foodsDiary.getDiaryFoods()
+  }
+
   $('#submit-food').on('click', event => {
     event.preventDefault()
     newFoodSequence()
   })
+
   $('#food-table-info').on('click', event => {
     if (event.target.nodeName == 'BUTTON') {
       var id = event.target.id.split('-')[2]
       foodsRequests.deleteFood(id)
       foodsRequests.removeFoodRow()
     }
   })
+
+  $('#food-table-info').on('focusout', e => {
```

Comment:
Curious about the 'e' here. Does it recognize that 'e' represents 'event', or can any variable be used here?

Reply:
I meant to change it to event for consistency's sake. Overall, I think any variable could be used here.

I just made this change and committed it.